tests/desktop_dinosaurs.pm

```diff
@@ -8,8 +8,7 @@ sub run {
     my $app_id = 'com.endlessm.dinosaurs.en';
 
     check_desktop_clean();
-    $self->add_nightly_sdk_repo();
-    $self->install_app($app_id, run_with_nightly_sdk => 1);
+    $self->install_app($app_id);
```

Comment:
This is only true, I guess, because right now the nightly and released SDKs are very close together. If/when development begins again on the SDK, it would seem sensible to test both cases.

Reply:
I’m fairly against doing that. We should be testing the images as they are generated; that’s the whole point of OpenQA. Why do we not use the nightly SDK by default on our nightly EOS master images?